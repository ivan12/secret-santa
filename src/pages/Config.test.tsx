import { render } from '@testing-library/react';
import React from 'react';
import { RecoilRoot } from 'recoil';
import Configuration from './Config';

const mockNavigation = jest.fn();

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavigation,
    };
});

describe('the configuration page', () => {
    test('should be rendered correctly', () => {
        const { container } = render(
            <RecoilRoot>
                <Configuration />
            </RecoilRoot>
        );

        expect(container).toMatchSnapshot();
    });
});
