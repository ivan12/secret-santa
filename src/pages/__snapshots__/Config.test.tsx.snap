// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the configuration page should be rendered correctly 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Let's get started! (Min 3 people)
      </h2>
      <form>
        <div
          class="input-btn-group"
        >
          <input
            class="input-name"
            maxlength="30"
            placeholder="Name"
            type="text"
            value=""
          />
          <input
            class="input-email"
            maxlength="100"
            placeholder="Email"
            type="email"
            value=""
          />
          <button
            disabled=""
          >
            Add
          </button>
        </div>
        
      </form>
      <footer
        class="footer-settings"
      >
        <button
          class="button"
          disabled=""
        >
          Go to the draw
        </button>
        <img
          alt="Shopping bags"
          src="/images/bags.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
